doc/ref/configuration/minion.rst

```diff
@@ -3219,3 +3219,31 @@ URL of the repository:
 Replace ``<commit_id>`` with the SHA1 hash of a commit ID. Specifying a commit
 ID is useful in that it allows one to revert back to a previous version in the
 event that an error is introduced in the latest revision of the repo.
+
+``ssh_merge_pillar``
+----------------------
```

Comment:
Can you delete the last two dashes here? Sphinx will warn on the docs build when the dashed line is longer than the line above it.